src/hooks/useMongo.js

```diff
@@ -16,7 +16,7 @@ import {useLogger} from "./useLogger";
  */
 export const useMongo = (input) => {
 
-    const logger = useLogger();
+    const loggingContext = useLogger();//useContext(LoggingContext).logger;
```

Comment:
I'm guessing that you are having issues accessing context from the useMongo hook? Because the hook by itself isn't mounted into the DOM before you instantiate it and pass it to the context provider I'm not sure this will work. I think probably the easiest way to pass a reference to our logger instance would be to just include an (optional) param to the useMongo hook like 
```
export const useMongo = (input, logger) => {...}
```

Reply:
I'll try that out tomorrow 👍